### PR TITLE
Fix openContainingFolder()

### PR DESCRIPTION
**Pull request by MadeOfMagicAndWires**
Hey, the code from [your pull](https://github.com/z411/trackma/pull/208) checked out but it just turned out to be an innocent importing error.

I also added some little things like moving the menu items to a more appropriate spot error escapes, and a more compatible way of opening the directories, but just keep what you like.

Probably are going to have to make a new pull in order to keep the commit log clean.

### CODE REVIEW

File: trackma/ui/gtkui.py [hunk 1]
```diff
@@ -148,14 +149,14 @@ def start(self, account):
         mb_show.append(self.mb_play)
         mb_show.append(self.mb_info)
         mb_show.append(self.mb_web)
+        mb_show.append(self.mb_folder)
```
Comment (MadeOfMagicAndWires) [inline trackma/ui/gtkui.py]: Move the menu item up

File: trackma/ui/gtkui.py [hunk 2]
```diff
@@ -742,12 +743,19 @@ def task_update_next_response(self, widget, response, show, played_ep):
...
-        show = self.engine.get_show_info(self.select_show)
+    def task_openContainingFolder(self):
+        
+        #get needed show info
+        show = self.engine.get_show_info(self.selected_show)
         titles = self.engine.data_handler.get_show_titles(show)
         filename, ep = self.engine._search_video(titles, 1)
 
-        currentFolderProcess = subprocess.Popen(["/bin/nautilus", filename])
+        try:
```
Comment (MadeOfMagicAndWires) [inline trackma/ui/gtkui.py]: `"/usr/bin/xdg-open"` with `os.path.dirname(filename)` allows this to work with whatever file manager the user has as default.

the tryblock is just from engine.play_episode()

File: trackma/ui/gtkui.py [hunk 3]
```diff
@@ -1122,11 +1130,11 @@ def showview_context_menu(self, treeview, event):
 
                 menu.append(mb_info)
                 menu.append(mb_web)
+                menu.append(mb_folder)
```
Comment (MadeOfMagicAndWires) [inline trackma/ui/gtkui.py]: Same as above, just moving the menu item to a more appropriate place

File: trackma/ui/gtkui.py [hunk 4]
```diff
@@ -32,6 +32,7 @@
 
 import webbrowser
 import os
+import subprocess
```
Comment (MadeOfMagicAndWires) [inline trackma/ui/gtkui.py]: the killer.

**Comment by anjandev** [comment]
Just a quick question, my hard drives got wiped and I dont have any shows to test this with. Does this work as intended?

**Comment by MadeOfMagicAndWires** [comment]
Yeah, It should work wherever trackma can find local files. Good of you too ask though, since I forgot to add an error message if no local files could be find, so please see #2

**Pull request merged**